### PR TITLE
stop importing from deprecated easybuild.tools.py2vs3 module + stop testing with Python 2.7 and 3.5

### PR DESCRIPTION
**Pull request by branfosj**
* [x] https://github.com/easybuilders/easybuild-framework/pull/4229

### CODE REVIEW

File: easybuild/easyblocks/generic/pythonpackage.py [hunk 1]
```diff
@@ -47,11 +47,11 @@
 from easybuild.framework.easyconfig.default import DEFAULT_CONFIG
 from easybuild.framework.easyconfig.templates import TEMPLATE_CONSTANTS
 from easybuild.framework.extensioneasyblock import ExtensionEasyBlock
+from easybuild.tools.asyncprocess import subprocess_popen_text
```
Comment (branfosj) [inline easybuild/easyblocks/generic/pythonpackage.py:50]:
```suggestion
from easybuild.tools.run import subprocess_popen_text
```

File: easybuild/easyblocks/u/ucx_plugins.py [hunk 2]
```diff
@@ -32,9 +32,9 @@
 import os
 
 from easybuild.easyblocks.generic.configuremake import ConfigureMake
+from easybuild.tools.asyncprocess import subprocess_popen_text
```
Comment (branfosj) [inline easybuild/easyblocks/u/ucx_plugins.py:35]:
```suggestion
from easybuild.tools.run import subprocess_popen_text
```

**Review by branfosj — COMMENTED** [review]
changed location of `subprocess_popen_text`

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**